scality_manila_utils/exceptions.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2015 Scality
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class DeserializationException(Exception):
+    pass
+
+
+class ExportException(Exception):
+    pass
+
+
+class EnvironmentException(Exception):
```

Comment:
There's a default `EnvironmentError`, is this similar? Can it be reused instead of introducing a new exception type?